lib/block_view_message.go

```diff
@@ -825,6 +863,101 @@ func (bav *UtxoView) _connectMessagingGroup(
 		}
 	}
 
+	// V3 Messages: MUTING and UNMUTING members
+	// Check if blockHeight lower than threshold and still trying to mute
+	if blockHeight < bav.Params.ForkHeights.DeSoV3MessagesMutingAndPrefixOptimizationBlockHeight {
```

Comment:
nit: how about we consolidate and refactor this `if existingEntry != nil && !existingEntry.isDeleted` -> `blockHeight < bav.Params...` || `blockHeight >= bav params....`?

Reply:
done